MODULE.bazel

```diff
@@ -70,6 +70,12 @@ use_repo(
     "tart",
 )
 
+rules_ios_bazel_version_deps = use_extension("//rules:module_extensions.bzl", "rules_ios_bazel_version_deps")
+use_repo(
+    rules_ios_bazel_version_deps,
+    "rules_ios_bazel_version",
+)
+
```

Comment:
Can we instead use: https://github.com/bazelbuild/bazel-skylib/blob/main/docs/versions_doc.md

Reply:
I added a comment explaining why we need it and why skylib versions won't work for us